app/helpers/orders_helper.rb

```diff
@@ -0,0 +1,22 @@
+module OrdersHelper
+  def into_money order_detail
+    price = order_detail.service.price
+    count_day = count_day(order_detail)
+    into_money = count_day * price
+
+    into_money
+  end
+
+  def total_money order
+    total_money = 0
```

Comment:
```suggestion
    order.order_details.reduce(0) do |sum, o|
      sum + o.into_money
    end
```
hàm này xử lý thế này thôi cũng được này